### PR TITLE
fix(viewer): Fix 3D viewer menu layout too narrow

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mxiao6 has signed the CLA. Thanks for the pull request!

File: src/lib/viewers/box3d/model3d/Model3D.scss [hunk 1]
```diff
@@ -18,6 +18,7 @@ $arrow-top-bottom: 4px solid transparent;
 
     .bp-pullup {
         bottom: 100%;
+        width: fit-content;
```
Comment (mickr) [inline src/lib/viewers/box3d/model3d/Model3D.scss]: Does this only affect Chrome? IE 11 doesn't support this AFAIK

Comment (mxiao6) [reply src/lib/viewers/box3d/model3d/Model3D.scss]: I tested IE 11 and Edge, and it looks fine even before the change.

Comment (jstoffan) [reply src/lib/viewers/box3d/model3d/Model3D.scss]: I don't think we should use experimental CSS values. Does setting `white-space: nowrap;` on the parent work?

**Pull request merged**